http_service/bugbug_http/app.py

```diff
@@ -407,6 +448,84 @@ def model_prediction(model_name, bug_id):
     return compress_response(data, status_code)
 
 
+@application.route(
+    "/<model_name>/predict/github/<string:owner>/<string:repo>/<int:issue_num>"
+)
+@cross_origin()
+def model_prediction_github(model_name, owner, repo, issue_num):
```

Comment:
Decided to create a separate route instead of adding a parameter to `model_prediction`, as I realized that owner and github repository needs to be specified as well